src/Compilers/Core/Portable/Operations/IFlowCaptureOperation.cs

```diff
@@ -8,8 +8,6 @@ namespace Microsoft.CodeAnalysis.FlowAnalysis
     /// </summary>
     public interface IFlowCaptureOperation : IOperation
     {
-        // PROTOTYPE(dataflow): Finalize the design how capturing/referencing intermediate results is represented.
```

Comment:
We decided to use `CaptureId`. @AlekseyTs let me know if this comment was capturing more work.